frontend/src/components/ShootMessageDetails.vue

```diff
@@ -68,13 +68,15 @@ SPDX-License-Identifier: Apache-2.0
             <div v-for="(lastErrorDescription, index) in errorDescriptions" :key="index">
               <v-divider v-if="index > 0" class="my-2"></v-divider>
               <v-alert
-                v-for="({description, userError, infraAccountError}) in lastErrorDescription.errorCodeObjects" :key="description"
-                color="error"
+                v-for="({description, hint, userError, infraAccountError}) in lastErrorDescription.errorCodeObjects" :key="description"
+                type="error"
                 :icon="userError ? 'mdi-account-alert' : 'mdi-alert'"
                 :prominent="!!userError"
               >
-                <h4 v-if="userError">Action required</h4>
-                <span class="wrap">
+                <h3 v-if="userError">Your Action is required</h3>
+                <h4 class="wrap-text font-weight-bold">This error is flagged as user error which indicates that no Gardener operator action is required.
+                Please read the error message carefully and take action.</h4>
```

Comment:
<img width="1006" alt="Screenshot 2022-05-09 at 17 26 39" src="https://user-images.githubusercontent.com/5526658/167443742-387d5615-5a7e-4caf-8210-b9288ad31e68.png">

> Please read the error message ..
> Please double-check the error message ..

I understand that the second message is coming from gardener, but it's duplicative in this case

Reply:
Yes, I understand your thoughts. However, for the other user errors there are completely different error messages and it is not duplicative. So I'm not sure what we should do here...

Reply:
I guess we should update the message in gardener then